Scripts/git-split.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Nit, but I would name this file `git-copy.sh` since that's what it actually does, and add comments describing the workflow of running this and THEN removing duplicate code, so the overall change is a split.

Reply:
`git-cp.sh`, in fact, since that's what the `Usage` line below refers to?